### PR TITLE
Pam=268 file upload and media types

### PR DESCRIPTION
**Pull request by pamgoodrich**
This request is pretty specific, but also useful for customers, I think. I created a "Work with Media Types" topic that we can hopefully build on over time, and if you have suggestions, I can add more types now to help make this topic more useful.

If you think I should stick to a "Work with File Uploads" topic, I can do that too. Let me know and happy to meet by phone for a few minutes if that helps.

Rendered version: https://meta.stoplight.io/docs/platform/branches/pam%3D268-file-upload-and-media-types/ZG9jOjUwNTQ1MDk5-work-with-media-types

### CODE REVIEW

**Review by daniel-white — CHANGES_REQUESTED** [review]
some small nits, some bigger things we should discuss

File: toc.json [hunk 1]
```diff
@@ -48,6 +48,11 @@
                 "title":"Shared Components",
                 "uri":"3.-design/shared-components.md"
              },
+             {
+                "type":"item",
+                "title":"Work with Content Types",
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply toc.json]: Changed to Media Types in the TOC so it matches the topic.

File: 3.-design/work-with-content-types.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Work with Media Types
+
+[Media types](https://swagger.io/docs/specification/media-types/) are a format of a request or response body data. 
+
+## Upload Files
+
+Use the multiform/form-data media type to upload files with request content for POST, PUT, and PATCH operations.
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply 3.-design/work-with-content-types.md]: Ugh. Sorry about that. Fixed.

File: 3.-design/work-with-content-types.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+              }
+            },
+````
+
+### Try It and Mocking
+In Edit mode: Select **Preview**, and then select **Try It**. You can then select **Upload** to navigate to a file. 
+
+![multiform-try-it.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/HqRH9m8wLmY)
+
+From published documentation, you can send a request to a live server or a mock server. Mock servers currently do not return valid requests for multiform/form-data media types, however.
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply 3.-design/work-with-content-types.md]: Done

**Comment by pamgoodrich** [comment]
Rendered version: https://meta.stoplight.io/docs/platform/branches/pam%3D268-file-upload-and-media-types/c2f20944173d7-work-with-media-types

File: 3.-design/work-with-content-types.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+[Media types](https://spec.openapis.org/oas/v3.1.0#media-types) are a format of a request or response body data. Each media type includes a schema that indicates the type of data and properties. 
+
+The Studio Form Designer provides a set of media types that you can easily add to request and response bodies. The default is application/json.
+
+![media-type.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/mrrzkTS703E)
+
+If a format you need is not listed, you can manually add it to the list.
+
+You can add multiple media types to request and response bodies as needed. The following example shows a request body with three media types: application/json, application/pdf, and text/html.
```
Comment (daniel-white) [inline 3.-design/work-with-content-types.md:11]:
```suggestion
Within a single operation, you can add multiple media types to request and response bodies as needed. Each media type indicates that the body contains appropriate content.  The following example shows a request body with three media types: `application/json`, `application/pdf`, and `text/html`. This indicates that the operation accepts one a JSON document, a PDF file or a HTML document.
```

File: 3.-design/work-with-content-types.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+# Work with Media Types
+
+[Media types](https://spec.openapis.org/oas/v3.1.0#media-types) are a format of a request or response body data. Each media type includes a schema that indicates the type of data and properties.
```
Comment (daniel-white) [inline 3.-design/work-with-content-types.md:3]:
```suggestion
[Media types](https://spec.openapis.org/oas/v3.1.0#media-types) declare the format of a request or response body content. For instance: 
- An endpoint on the server allow accepting PDF files with `application/pdf`, for a file upload
- An endpoint may return a PNG image with `image/png` for display on a web page
- A client and server may exchange messages with `application/json` or `application/xml`
```

File: 3.-design/work-with-content-types.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+![media-type.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/mrrzkTS703E)
+
+If a format you need is not listed, you can manually add it to the list.
+
+You can add multiple media types to request and response bodies as needed. The following example shows a request body with three media types: application/json, application/pdf, and text/html.
+
+![Mutliple Media Types](https://stoplight.io/api/v1/projects/cHJqOjI/images/aYpG8hvuopc)
+
+Here is the JSON for this example:
```
Comment (daniel-white) [inline 3.-design/work-with-content-types.md:15]:
```suggestion
Here is a snippet of an Open API spec in JSON for this example:
```

File: 3.-design/work-with-content-types.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+                "type": "object",
+                "properties": {}
+              }
+            }
+          }
+        }
+      },
+```     
+
+## Multipart/form-data
```
Comment (daniel-white) [inline 3.-design/work-with-content-types.md:43]:
```suggestion
## Working with multipart and form requests
```

File: 3.-design/work-with-content-types.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
+# Work with Media Types
+
+[Media types](https://spec.openapis.org/oas/v3.1.0#media-types) are a format of a request or response body data. Each media type includes a schema that indicates the type of data and properties. 
+
+The Studio Form Designer provides a set of media types that you can easily add to request and response bodies. The default is application/json.
+
+![media-type.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/mrrzkTS703E)
+
+If a format you need is not listed, you can manually add it to the list.
```
Comment (daniel-white) [inline 3.-design/work-with-content-types.md:9]:
```suggestion
If a media type you need is not listed, you can manually add it to the list.
```

File: 3.-design/work-with-content-types.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+- An endpoint may return a PNG image with `image/png` for display on a web page.
+- A client and server may exchange messages with `application/json` or `application/xml`.
+
+The Studio Form Designer provides a set of media types that you can easily add to request and response bodies. REST APIs designed with Open API typically use `application/json` for communicating messages in the request and response bodies, but you are not required to do so.
+
+![media-type.png](https://stoplight.io/api/v1/projects/cHJqOjI/images/mrrzkTS703E)
+
+If a media type you need is not listed, you can manually add it to the list.
+
+Within a single operation, you can add multiple media types to request and response bodies as needed. Each media type indicates that the body contains appropriate content. The following example shows a request body with three media types: `application/json`, `application/pdf`, and `text/html`. This indicates that the operation accepts one a JSON document, a PDF file or a HTML document.
```
Comment (daniel-white) [inline 3.-design/work-with-content-types.md:16]:
```suggestion
Within a single operation, you can add multiple media types to request and response bodies as needed. Each media type indicates that the body contains appropriate content. The following example shows a request body with three media types: `application/json`, `application/pdf`, and `text/html`. This indicates that the operation accepts either a JSON document, a PDF file or a HTML document in a single request.
```

Comment (pamgoodrich) [reply 3.-design/work-with-content-types.md]: Last comment added.

File: 3.-design/work-with-content-types.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+                "type": "object",
+                "properties": {}
+              }
+            }
+          }
+        }
+      },
+```     
+
+## Work with Multipart abd Form Requests
```
Comment (mnaumanali94) [inline 3.-design/work-with-content-types.md:48]:
```suggestion
## Work with Multipart and Form Requests
```

File: 3.-design/work-with-content-types.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "file": {
+                    "type": "string",
+                    "format": "binary"
+                  }
+                }
+              }
+            },
```
Comment (mnaumanali94) [inline 3.-design/work-with-content-types.md:84]:
```suggestion
            }
```

**Review by mnaumanali94 — APPROVED** [review]

**Review by daniel-white — APPROVED** [review]

**Pull request merged**